### PR TITLE
Fixes issue with many to many relationship not respecting renamed tables

### PR DESCRIPTION
**Pull request by joaommartins**

### CODE REVIEW

File: butane/Cargo.toml [hunk 1]
```diff
@@ -26,17 +26,14 @@ uuid = ["butane_core/uuid", "butane_codegen/uuid"]
 
 [dependencies]
 butane_codegen = { path = "../butane_codegen", version = "0.5" }
-butane_core = { path = "../butane_core", version = "0.5" }
+butane_core = { path = "../butane_core", version = "0.5", features = ["sqlite"] }
```
Comment (jayvdb) [inline butane/Cargo.toml]: this shouldnt be necessary.

maybe you need sqlite for the tests?  if so, you may be able to add it as a `dev-dependencies` by using https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml

File: butane/Cargo.toml [hunk 2]
```diff
@@ -26,17 +26,14 @@ uuid = ["butane_core/uuid", "butane_codegen/uuid"]
...
 
 
 [dev-dependencies]
 cfg-if = "1.0"
 exec_time = { version = "0.1.4" }
 paste = "1.0.11"
-chrono = { version = "0.4", default-features = false, features = [
```
Comment (jayvdb) [inline butane/Cargo.toml]: this was intentional ; see https://github.com/Electron100/butane/pull/47

File: butane_core/src/many.rs [hunk 3]
```diff
@@ -18,7 +18,7 @@ fn default_oc<T>() -> OnceCell<Vec<T>> {
 /// many-to-many relationship with U, owner type is T::PKType, has is
 /// U::PKType. Table name is T_ManyToMany_foo where foo is the name of
 /// the Many field
-//
+// TODO: Comment is OOD
```
Comment (jayvdb) [inline butane_core/src/many.rs]: probably want to resolve this before the PR upstream

File: butane/Cargo.toml (hunk 1 above)
Comment (joaommartins) [reply butane/Cargo.toml]: Reverted.

File: butane_core/src/many.rs (hunk 3 above)
Comment (joaommartins) [reply butane_core/src/many.rs]: Fixed.

File: butane/Cargo.toml (hunk 2 above)
Comment (joaommartins) [reply butane/Cargo.toml]: Reverted.

File: butane_core/src/codegen/dbobj.rs [hunk 4]
```diff
@@ -342,12 +342,17 @@ where
         .collect()
 }
 
-fn many_table_lit(ast_struct: &ItemStruct, field: &Field) -> LitStr {
-    let tyname = &ast_struct.ident;
+fn many_table_lit(ast_struct: &ItemStruct, field: &Field, config: &Config) -> LitStr {
+    // let tyname = &ast_struct.ident;
```
Comment (jayvdb) [inline butane_core/src/codegen/dbobj.rs]: delete this?

**Review by jayvdb — APPROVED** [review]
one minor query  , but it is not a blocker for merging to our fork.

File: butane_core/src/codegen/dbobj.rs (hunk 4 above)
Comment (joaommartins) [reply butane_core/src/codegen/dbobj.rs]: Fixed it, thanks.

**Pull request merged**